### DISCUSSION TITLE
show_sun_elevation: false

### DISCUSSION THREAD

**Discussion by LloydOS** [jekhokie/raspberry-noaa-v2#234] [Q&A] [answered]
Hi
After editing the 
Updating the setting
`sudo nano config/settings.yml`
`show_sun_elevation: false`
When viewing the captured images, it still displays the Sun Elevation
Any suggestions?
![NOAA-19-20210308-081454-MCIR](https://user-images.githubusercontent.com/80271054/110341640-78458f00-8022-11eb-9300-7eaae3c76d5e.jpg)

**Comment by jekhokie** [comment]
Hi @LloydOS - if you're using version 1.4 or later of the framework (https://github.com/jekhokie/raspberry-noaa-v2/releases/tag/v1.4.0), there are a couple of parameters that are no longer applicable, including the sun elevation. To adjust your annotation on the images, you'll now want to edit the `config/annotation/annotation.html.j2` file according to the directions in the annotation document here: https://github.com/jekhokie/raspberry-noaa-v2/blob/master/docs/annotation.md

Also, check out the release notes in version 1.4 (https://github.com/jekhokie/raspberry-noaa-v2/releases/tag/v1.4.0) - they indicate you should remove those parameters from the `config/settings.yml` file.

**Reply by LloydOS** [reply]
Hi @jekhokie - Thanks for the response
How do I find out what version I am running? Only installed it in the last 10 days.
I checked the `config/annotation/` folder and it is empty
`pi@RPiWXsat:~/config/annotation $ cd ~/config/annotation
pi@RPiWXsat:~/config/annotation $ ls -lah
total 8.0K
drwxr-xr-x 2 pi pi 4.0K Mar  7 16:41 .
drwxr-xr-x 3 pi pi 4.0K Mar  7 20:39 ..
pi@RPiWXsat:~/config/annotation $`

Should the
`
./install_and_upgrade.sh`
have populated that?

Thanks

LloydOS

**Comment by jekhokie** [comment, marked as answer]
@LloydOS I just cut version 1.5.0 - my suggestion would be to upgrade to latest by doing the following:

```
git pull
# edit config/settings.yml to include new settings from config/settings.yml.sample
./install_and_upgrade.sh
```

You should then have the annotation template in place that you can edit for future captures.

**Reply by LloydOS** [reply]
After doing the last update I now have the files to edit `config/annotation/annotation.html.j2`
Thanks for the information.
![NOAA-19-20210311-0739-MCIR-precip](https://user-images.githubusercontent.com/80271054/110910440-3dd14000-8309-11eb-9b9b-7124f7d54f8f.JPG)

**Reply by LloydOS** [reply]
Where are the images stored on the Pi ?

**Reply by jekhokie** [reply]
@LloydOS in `/srv/images/`

**Reply by LloydOS** [reply]
Thanks for your assistance as always.
